elasticsearch/Dockerfile.centos7

```diff
@@ -43,7 +43,8 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
                 PyYAML && \
     yum clean all
 
-
+ENV OSE_ES_URL=file:///openshift-elasticsearch-plugin-2.4.4.13-SNAPSHOT.zip
+ADD openshift-elasticsearch-plugin-2.4.4.13-SNAPSHOT.zip /
```

Comment:
I'm assuming this is only for testing purposes and will be removed from the final PR before merging

Reply:
grr...